### DISCUSSION TITLE
[Kavita] FanFiction Support

### DISCUSSION THREAD

**Discussion by lapidyll** [Kareadita/Kavita#2632] [Ideas]
### Idea Description

I would like to see fanfiction supported. 

In Calibre I use a plugin called [FanFicFare ](https://github.com/JimmXinu/FanFicFare) to import fanfics from websites, then tag them with custom metadata to sort by their fandom. In Moon Reader it creates a custom library that I can open and then select a fandom which then has the associated fics. 

So my folder structure looks like: Fanfics > Star Trek > Multiple Individual epubs.

In Kavita I can sort of get this same functionality if I tag each fic as a Series instead, but each fic gets labeled as a part rather than separate entries. One fanfic becomes 1.0: FanFic Title, another becomes 2.0: FanFic Title, etc. I'd prefer they were just their titles and 'unrelated' except for their 'fandom' tag.

### Duration of Using Kavita

1 month

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by scare376** [comment]
I'm confused by what you are asking for here. If you are tagging them then why not just have the fandom as a tag and then create a smart filter that will only show books with that tag.

**Reply by DieselTech** [reply]
Also agree here, the tags could be used to create smart filters that can handle this type of setup without any additional overhead in terms of programming and maintenance.

**Reply by lapidyll** [reply]
Hello! I see what you're saying and understand the confusion. In Calibre I had created an entirely new field in the metadata to separate everything. The original tags in Calibre were being populated with tags provided the website I grab the fics from. I tried clearing tags in one and just gave it a fandom tag, but it doesn't seem to appear in Kavita even after scanning the library for changes? I try to filter by tag but there's no option available in the dropdown.

Edit: I figured out what's happening. For some reason, my calibre tags end up under genre in Kavita.

**Reply by scare376** [reply]
Then this is more of a bug then a feature request and I believe there is already a report about epub tags not appearing.

**Reply by majora2007** [reply]
@lapidyll please create an issue and I can work with you on getting the bug sorted, then we can close this discussion.
